developer_docs/how_to_create_a_plugin.asciidoc

```diff
@@ -63,18 +63,18 @@ contribution guide] describes setting up a small test instance.
 
 You can enable the plugin right away, and see what it's default
 behavior is, by editing foreman Gemfile.local.rb file (or creating this
-file under the folder bundler.d) and adding the following line
+file under the folder /usr/share/foreman/bundler.d) and adding the following line
 
 .Gemfile.local.rb
 [source, ruby]
 ----
 gem 'my_plugin', :path => 'path_to/my_plugin'
 ----
-Install the 'preface' bundle by running from foreman core directory:
+Install the 'preface' bundle by running from foreman core directory (/usr/share/foreman):
 
 [source, bash]
 ----
-bundle install
+BUNDLE_GEMFILE=Gemfile.in bundle install
```

Comment:
This isn't quite right.  It's Gemfile.in in the this dir, but Gemfile in a plugin dir and it complains about that.